spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -16,165 +16,141 @@
+
+      # click update address link
+      click_link t('idv.buttons.change_address_label')
+      expect(page).to have_content(t('in_person_proofing.headings.update_address'))
+      fill_in t('idv.form.address1'), with: '987 Fake St.'
+      click_button t('forms.buttons.submit.update')
+      expect(page).to have_content(t('headings.verify'))
+      expect(page).to have_current_path(idv_in_person_verify_info_path)
+      expect(page).to have_text('987 Fake St.')
+      expect(page).not_to have_text('bad address')
+
+      # click update ssn button
+      click_on t('idv.buttons.change_ssn_label')
+      expect(page).to have_content(t('doc_auth.headings.ssn_update'))
+      fill_in t('idv.form.ssn_label'), with: '900-12-2345'
+      click_button t('forms.buttons.submit.update')
+      expect(page).to have_content(t('headings.verify'))
+      expect(page).to have_current_path(idv_in_person_verify_info_path)
+      expect(page).to have_text('9**-**-***5')
+    end
   end
 
-  it 'does not proceed to the next page if resolution fails',
-     allow_browser_log: true do
-    sign_in_and_2fa_user
-
-    begin_in_person_proofing(user)
-    complete_prepare_step(user)
-    complete_location_step(user)
-    complete_state_id_controller(user)
-    fill_out_ssn_form_with_ssn_that_fails_resolution
-    click_idv_continue
-    complete_verify_step(user)
-
-    expect(page).to have_current_path(idv_session_errors_warning_path(flow: 'in_person'))
-    click_on t('idv.failure.button.warning')
-
-    expect(page).to have_current_path(idv_in_person_verify_info_path)
+  context 'when completing the verify info step' do
+    context 'when SSN resolution fails' do
+      it 'navigates to the session warning page' do
+        sign_in_and_2fa_user(user)
+        begin_in_person_proofing(user)
+        complete_prepare_step(user)
+        complete_location_step(user)
+        complete_state_id_controller(user)
+        fill_out_ssn_form_with_ssn_that_fails_resolution
+        click_idv_continue
+        complete_verify_step(user)
+
+        expect(page).to have_current_path(idv_session_errors_warning_path(flow: 'in_person'))
+        click_on t('idv.failure.button.warning')
+
+        expect(page).to have_current_path(idv_in_person_verify_info_path)
+      end
+    end
+
+    context 'when SSN resolution passes' do
+      before do
+        complete_steps_before_info_verify(user)
+        complete_verify_step(user)
+      end
+
+      it 'navigates to the verify phone' do
+        expect(page).to have_content(t('doc_auth.forms.doc_success'))
+        expect(page).to have_content(t('titles.idv.phone'))
+        expect(fake_analytics).to have_logged_event(
+          'IdV: doc auth verify proofing results',
+          hash_including(analytics_id: 'In Person Proofing'),
+        )
+      end
+    end
   end
 
-  it 'proceeds to the next page if resolution passes',
-     allow_browser_log: true do
-    sign_in_and_2fa_user
+  def complete_steps_before_info_verify(user)
```

Comment:
`spec/support/features/in_person_helper.rb` includes the `complete_all_in_person_proofing_steps` method and `spec/support/features/doc_auth_helper.rb` includes the `complete_up_to_how_to_verify_step_for_opt_in_ipp` method, both of which are somewhat similar to this method.  What would you think of moving this method to the `in_person_helper`?

Reply:
I can do that. That will be a good way to dry things up.